README.md

```diff
@@ -1 +1,65 @@
-# cli-recommendation
\ No newline at end of file
+# Brand New AZ Interactive
+We are very excited to announce that Azure CLI team combined AI with CLI interactive mode *(az interactive)* to provide users with an intelligent interactive experience.
+
+It is mainly oriented to the inexperienced users in interactive mode to help them reduce the learning cost, provide an intelligent interactive experience, and improve the operation efficiency in some complex scenarios.
+
+
+## What is it
+
+Currently, our product mainly provides customers with four revolutionary features:
+
+• Intelligent Completion
+
```

Comment:
Would it be better if we put the content of images, README, and videos in the CLI extension repo?